### DISCUSSION TITLE
Unable to update Open WebUI to v0.3.8 from v0.3.10 on Docker

### DISCUSSION THREAD

**Discussion by CodeGrafus** [open-webui/open-webui#4012] [General]
Unable to update Open WebUI to v0.3.8 from v0.3.10 on Docker
I'm having trouble updating my Open WebUI installation from version v0.3.8 to v0.3.10 on Docker. I've tried the following steps:

Deleted the existing Open WebUI image and container using docker rm and docker rmi commands.
Pulled the latest Open WebUI image using docker pull ghcr.io/open-webui/open-webui:v0.3.10.
Created a new container using the updated image.
However, the new container is still running Open WebUI version v0.3.8. I've also tried installing Open WebUI from scratch using the latest image, but it still installs version v0.3.8.

I've checked the Open WebUI documentation and followed the update instructions carefully, but I'm still unable to update the installation.

**Comment by loki6000** [comment]
same issues with the new one, deleted and local and pulled latest. still see the message A new version (v0.4.1) is now available. [Update for the latest features and improvements.](https://github.com/open-webui/open-webui/releases)

**Comment by DIE020HARDZ** [comment]
I had this problem too but recently I found out that it's because the CUDA supported docker run command line is outdated so it runs with the old version.
Here is the command that line that they give you in their official page : docker run -d -p 3000:8080 --gpus all -v open-webui:/app/backend/data --name open-webui ghcr.io/open-webui/open-webui:cuda
Here is what I made change : docker run -d -p 3000:8080 --gpus all -v open-webui:/app/backend/data --name open-webui-nvidia ghcr.io/open-webui/open-webui:0.4.7-cuda 
(latest version at this time)
(The old command line only run Open WebUI in a v.3.35 with CUDA supported version.)

**Reply by DIE020HARDZ** [reply]
You can find their latest Open WebUI package in here : https://github.com/open-webui/open-webui/pkgs/container/open-webui

**Reply by cbri321** [reply]
This just helped me.  I've been using :latest and it wasn't grabbing the correct image.  Thanks!
